include/option.h

```diff
@@ -0,0 +1,29 @@
+#define MAX_OPT_LEN 6
+
+typedef enum{
+    FALSE,
+    TRUE,
+    ERRORBOOL,
+} bool_t;
+
+typedef enum{
+    ADDNID,
+    ADDBID,
+    RMNID,
+    RMBID,
+    LS,
+    SYNC,
+    QUIT,
+    NONE,
+    ERROROPT,
+} option_t;
+
+typedef struct input_split{
```

Comment:
should we simplify the name of this since we will be writing a lot to `input_t` ?